### ISSUE TITLE
训练代码scripts/diffusion/train.py的462行(1-sigma_min)*x_1-x_0的疑问

### ISSUE THREAD

**Issue by gongfc** [hpcaitech/Open-Sora#903]
您好：
       我有一个训练代码中，我有一处不理解的地方，还请解答！

(1). 我的问题： x_1 - x_0代表什么含义？
       在训练代码scripts/diffusion/train.py的462行
       "v_t = (1-sigma_min)*x_1-x_0" (在run_iter函数中)  : 
       x_1 : 应该是随机添加的噪声数据，
       x_0 : 应该是原始视频的编码数据，
       (参见train.py的432行，也能佐证x_0,x_1的含义！)
       那么, v_t代表什么？ 
(2). 我的分析：
       在上一行“model_pred = model(**inp)”应该是执行前向计算，得到视频数据中的噪声，
那么后面一行计算损失"loss=F.mse_loss(model_pred, v_t)"应该是预测噪声和真实噪声的损失？
        
感谢！！不知道我哪里理解得不正确，还请纠正 !
